app/src/main/java/com/xmartlabs/doapp/model/Task.java

```diff
@@ -29,15 +29,15 @@
   @PrimaryKey(autoincrement = true)
   long id;
   @Column
-  String name;
+  String title;
+  @Column
+  String description;
   @Column(typeConverter = LocalDateAdapter.class)
   LocalDate date;
   @Column
-  Double duration;
-  @Column
   boolean isFinished;
-  @ForeignKey
-  User user;
-  @ForeignKey
-  Group group;
+//  @ForeignKey
```

Comment:
Ok. In those cases add a comment saying that will be used soon.

It's a good practice to comment why the code is commented, as commented code is usually bad practice.